css-canvas-medical/styles.css

```diff
@@ -0,0 +1,282 @@
+* {
+  box-sizing: border-box;
+}
+
+header {
+  display: flex;
+  justify-content: end;
+  border-bottom: solid 1px lightgray;
+}
+
+#title {
+  margin-left: 50px;
+  margin-right: 900px;
+  font-family: "Tiempos Headline", serif;
+  color: navy;
+}
+
+li,
+p,
+h3 {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+}
+
+h1 {
+  font-family: "Tiempos Headline", serif;
+  font-size: 45px;
+  font-weight: lighter;
+  width: 500px;
+  padding-top: 85px;
+  margin-bottom: 0;
+}
+
+.top > p {
+  width: 555px;
+  font-size: 20px;
+  line-height: 30px;
+  padding-top: 0;
+  margin-bottom: 30px;
+}
+
+li {
+  display: inline-block;
+  margin: 20px 25px;
+  font-size: 20px;
+}
+
+.pages {
+  text-decoration: none;
+  color: #a2a2a2;
+}
+
+#blue-logo {
+  float: left;
```

Comment:
You should avoid using `float` at all costs. It also doesn't seem to impact anything if you remove it, so if it's not adding any additional functionality, there's no reason to keep it.